src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java

```diff
@@ -217,16 +218,18 @@ private Date calculateAgePreferencesToDate(int preference){
   /**
    * Gets a game from a random user, but not the game from the user calling himself, and neither a game that that user
    * already has played on
-   * @param user: user that makes the call
+   * @param userId: userId of the caller
    * @return Game: a random Game.
    * @throws ResponseStatusException - 404: if there is no game of another user left
    */
-  public Game getGameFromRandomUser(User user) {
+  public Game getGameFromRandomUser(Long userId, User user) {
```

Comment:
Why do you have an additional parameter userId? Should user.getId() not be the same as userId?

Reply:
done